tutorials/install-ubuntu-server/install-ubuntu-server.md

```diff
@@ -1,64 +1,164 @@
 ---
 id: tutorial-install-ubuntu-server
-summary: A very high-level guide on how to install Ubuntu Server for general use.
+summary: Turn your PC into a powerful server, capable of delivering anything from file sharing and local backup, to fully fledged web sites and beyond.
 categories: server
 tags: tutorial,installation,ubuntu,server,Ubuntu 16.04 LTS
 difficulty: 2
 status: Published
-published: 2017-05-31
+published: 2017-07-07
 
 ---
 
-# Installing Ubuntu Server for general use
+# Install Ubuntu Server
 
-## Getting started
-Duration: 0:05
+## Overview
+Duration: 0:01
 
-The basic steps to install Ubuntu Server from CD or USB stick are the same for installing any operating system. Unlike the desktop version, Ubuntu Server does not include a graphical installation program. Instead, it uses a console menu-based process.
+Ubuntu server is Ubuntu tailored for networks and services, and it's just as capable sharing files on your home network as it is running a 50,000 render farm.
```

Comment:
I took this example from https://www.ubuntu.com/server to present a unified message. But I agree, it's a little obscure. I'll change it.